data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -153,6 +153,32 @@
     "Percent of individuals < 100% Federal Poverty Line in 2010"
 )
 
+# RSEI Aggregated Micro-data
+EPA_RSEI_NUMBER_FACILITIES_OUTPUT_FIELD = (
```

Comment:
We don't use "OUTPUT_" in any of the other names in this file. Can you just call them "_FIELD"? E.g., `EPA_RSEI_NUMBER_FACILITIES_FIELD`